### PR TITLE
Fix: isEmpty() return true if it's empty

### PR DESCRIPTION
**Pull request by corentinheadoo**
The function isEmpty() return always false... fix it

### CODE REVIEW

File: src/Model/Translatable/TranslationMethods.php [hunk 1]
```diff
@@ -93,6 +93,13 @@ public function getLocale()
      */
     public function isEmpty()
     {
-        return false;
+        foreach (get_object_vars($this) as $var => $value) {
+            if (in_array($var, array('id', 'translatable', 'locale')))
```
Comment (cursedcoder) [inline src/Model/Translatable/TranslationMethods.php]: please use short syntax array

File: src/Model/Translatable/TranslationMethods.php [hunk 2]
```diff
@@ -93,6 +93,13 @@ public function getLocale()
...
+                continue;
+            
+            if (!empty($value))
+                return false;
```
Comment (cursedcoder) [inline src/Model/Translatable/TranslationMethods.php]: please use enclosing brackets `{}`

File: src/Model/Translatable/TranslationMethods.php [hunk 3]
```diff
@@ -93,6 +93,13 @@ public function getLocale()
...
+        }
+        return true;
```
Comment (cursedcoder) [inline src/Model/Translatable/TranslationMethods.php]: return statement should be separated by 1 space line

**Comment by Pierstoval** [comment]
LGTM 👍 /cc @cursedcoder

**Comment by docteurklein** [comment]
this method is meant to be overriden. e just provide a dummy impl to avoid extra work.
do we want a *smart* isEmpty?

**Pull request merged**